awx/api/views.py

```diff
@@ -187,6 +203,8 @@ def get(self, request, format=None):
         data['config'] = reverse('api:api_v1_config_view', request=request)
         data['settings'] = reverse('api:setting_category_list', request=request)
         data['me'] = reverse('api:user_me_list', request=request)
+        if get_request_version(request) > 1:
+            data['oauth'] = reverse('api:user_me_oauth_root_view', request=request)
```

Comment:
I'm super skeptical about having this on the root view. I feel like this is probably better serviced under the `me/` resource on the user object related resources displayed there if `user == me`.

Reply:
After thinking about this for a few minutes, I'm wondering if it might not be better to entirely move this to the top level of the api (inside of homing under the `me/`.

The view should be pretty limited to what the user can see/has access to anyway.

Reply:
We should remove or place a deprecation warning on the `authtoken` element above.